### PR TITLE
Added a card in Text Formatting - Text Tune

### LINKED ISSUE #2197: [Add] <A Card in Text Formatting>
### Site URL

https://text-tune.netlify.app/

### Description of the above provided site

Text tune is the ultimate solution for all the text 📖 formatting needs 🚀! The website provides text formatting features like UPPERCASE, lowercase, Title Case, Reverse Text, Remove WhiteSpace and another feature of this website is to provide with synonyms and antonyms for the word given by the user. If you are looking to format your text or get synonyms and antonyms for given words quickly, this website can be helpful 😀.  

Some images of the website:

Screenshot 2023-11-29 at 22 44 02
Screenshot 2023-11-29 at 22 44 10
Screenshot 2023-11-29 at 22 44 18
Screenshot 2023-11-29 at 22 44 34

### Category

Other

### Sub Category

Text Formatting

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by nelsonuprety1**
## Fixes Issue

Closes #2197

### CODE REVIEW

**Comment by nelsonuprety1** [comment]
@Anmol-Baranwal @CBID2 @rupali-codes

**Review by CBID2 — DISMISSED** [review]
LGTM

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
I don't see a subcategory with text-formatting.

image

You need to add files as per contributing guidelines, just adding a json file won't create a subcategory.

**Comment by nelsonuprety1** [comment]
Hi @Anmol-Baranwal thank you for the reply. So, should i create a new subcategory Text Formatting inside the Other category?

**Comment by nelsonuprety1** [comment]
@Anmol-Baranwal  I have fixed the issues and added the subcategory properly. Please kindly check this PR thank you.
Screenshot 2023-12-01 at 23 03 51

**Review by Anmol-Baranwal — DISMISSED** [review]
🚀 Looks good to me. 

Thanks for the changes :)

> @rupali-codes @aftabrehan @CBID2 Please review :)

File: database/other/text-formatting.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[{
+    "name": "Text Tune",
+    "description": "Text tune is the ultimate solution for all the text formatting needs!The website provides text formatting features like UPPERCASE, lowercase, Title Case, Reverse Text, Remove WhiteSpace and another feature of this website is to provide with synonyms and antonyms for the word given by the user. If you are looking to format your text or get synonyms and antonyms for given words quickly, this website can be helpful.",
```
Comment (aftabrehan) [inline database/other/text-formatting.json:3]:
```suggestion
    "description": "Text tune is the ultimate solution for all the text formatting needs! The website provides text formatting features like UPPERCASE, lowercase, Title Case, Reverse Text, Remove WhiteSpace and another feature of this website is to provide synonyms and antonyms for the word given by the user.",
```

> If you are looking to format your text or get synonyms and antonyms for given words quickly, this website can be helpful.

<details><summary>Why remove this?</summary>
This sentence isn't required, it just reference to the "synonyms and antonyms" feature again.



</p>
</details>

**Review by aftabrehan — CHANGES_REQUESTED** [review]
@nelsonuprety1, Thanks for the contribution. It could be useful.

Could you please help to adjust the text?

**Review by nelsonuprety1 — COMMENTED** [review]
I have adjusted the text as per you request. Thank You.

**Pull request merged**